KamerConnect.View.MAUI/Views/InterestsForm.xaml.cs

```diff
@@ -1,9 +1,86 @@
+using KamerConnect.Models;
+using KamerConnect.Services;
+using Npgsql;
+
 namespace KamerConnect.View.MAUI.Views;
 
 public partial class InterestsForm : ContentView
 {
-	public InterestsForm()
+	private readonly PersonService _personService;
+	private Person? _currentPerson;
+	private Personality? personality;
+	public Personality? Personality
+	{
+		get => personality;
+		set
+		{
+			personality = value;
+		}
+	}
+	
+	public InterestsForm(PersonService personService, Person person)
 	{
+		_personService = personService;
+		_currentPerson = person;
+		Personality = person.Personality;
```

Comment:
Personality zit al in person dit is dus een onnodige variabele. Je kan in het hele bestand als het goed is gewoon gebruik maken van person.Personality